### PR TITLE
[tools/onert_train] Add trainable ops support

### PR DESCRIPTION
**Pull request by mbencer**
This commits adds new flags of onert_train - trainable_ops_idx. It allows to pass trainable ops indexes nnfw training.

ONE-DCO-1.0-Signed-off-by: Mateusz Bencer <m.bencer@partner.samsung.com>

Issue: https://github.com/Samsung/ONE/issues/12386

### CODE REVIEW

**Comment by ragmani** [comment]
Could you update `README.md` with an example of executing `onert_train` with the added option?

**Comment by mbencer** [comment]
> Could you update `README.md` with an example of executing `onert_train` with the added option?

Sure, readme updated ;)

**Review by jyoungyun — APPROVED** [review]
LGTM Thank you!

**Review by zetwhite — APPROVED** [review]
LGTM :+1:

File: tests/tools/onert_train/src/onert_train.cc [hunk 1]
```diff
@@ -145,6 +145,13 @@ int main(const int argc, char **argv)
...
+    size_t pos = 0;
+    tri.trainble_ops_size = args.getTrainableOpsIdx().size();
+    for (auto const &idx : args.getTrainableOpsIdx())
+    {
+      tri.trainble_ops_idx[pos++] = idx;
+    }
+
     std::cout << "== training parameter ==" << std::endl;
     std::cout << tri;
     std::cout << "========================" << std::endl;
```
Comment (zetwhite) [inline tests/tools/onert_train/src/onert_train.cc:157]: (suggestion)

Printing `trainable_ops` would be helpful for the user.   

https://github.com/Samsung/ONE/blob/313fb05beae628308da4b843118330bc71a4a999/tests/tools/onert_train/src/nnfw_util.cc#L106-L112

**Review by ragmani — APPROVED** [review]
LGTM :+1:

**Pull request merged**